packages/cli/src/commands/models/m365LaunchInfo.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { CLICommand, err, ok } from "@microsoft/teamsfx-api";
+import { PackageService } from "@microsoft/teamsfx-core";
+import { getTokenAndUpn } from "../../cmds/m365/m365";
+import { logger } from "../../commonlib/logger";
+import { MissingRequiredOptionError } from "../../error";
+import { TelemetryEvent } from "../../telemetry/cliTelemetryEvents";
+import { sideloadingServiceEndpoint } from "./m365Sideloading";
+
+export const m365LaunchInfoCommand: CLICommand = {
+  name: "launchinfo",
+  description: "Get launch information of an acquired M365 App.",
+  options: [
+    {
+      name: "title-id",
+      description: "Title ID of the acquired M365 App.",
+      type: "string",
+    },
+    {
+      name: "manifest-id",
+      description: "Manifest ID of the acquired M365 App.",
+      type: "string",
+    },
+  ],
+  examples: [
+    {
+      command: "teamsfx m365 launchinfo --title-id U_xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
+      description: "Get launch information of the acquired M365 App by Title ID",
+    },
+    {
+      command: "teamsfx m365 launchinfo --manifest-id xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
+      description: "Get launch information of the acquired M365 App by Manifest ID",
+    },
+  ],
+  telemetry: {
+    event: TelemetryEvent.M365LaunchInfo,
+  },
+  handler: async (ctx) => {
+    logger.warning("This command is in preview.");
```

Comment:
The original one (https://github.com/OfficeDev/TeamsFx/blob/dev/packages/cli/src/cmds/m365/m365.ts#L80) sets log level to verbose to output detailed information.

Is it possible to ensure the same here?

Reply:
We will unify the log verbose by "--verbose" option. And we will add a new verbose() API in logProvider interface. This involved defining standards on what content should be classified as verbose. I will drive for this discussion.